.github/workflows/ci.yml

```diff
@@ -44,4 +44,6 @@ jobs:
       - uses: golangci/golangci-lint-action@v3.7.0
         with:
           version: v1.52.2
+          only-new-issues: true
+          skip-build-cache: true
```

Comment:
fixes an issue related to the linter spamming missing files if there is a linter problem on CI